Mark serverless environments as supported (#2895)
fastify/fastify: docs/Serverless.md

- [Vercel](#vercel)

### Attention Readers:

> The Fastify framework is designed to make implementing a traditional HTTP/S server easy.

> Serverless environments requests differently than a standard HTTP/S server and they have

Reviewer: ```suggestion
Serverless environments requests differ from a standard HTTP/S server and they have
```

different requirements. Nevertheless, Fastify supports serverless environments directly

> or through adapter. Be sure to test your application thoroughly as those difference might

Reviewer: ```suggestion
or through an adapter. Be sure to test your application thoroughly as those difference might
```
